ext/civicrm_admin_ui/managed/SavedSearch_Manage_Mail_Accounts.mgd.php

```diff
@@ -0,0 +1,194 @@
+        'name',
+      ],
+    ],
+  ],
+  [
+    'name' => 'SavedSearch_Mail_Accounts_SearchDisplay_Mail_Accounts_Table_1',
+    'entity' => 'SearchDisplay',
+    'cleanup' => 'always',
+    'update' => 'unmodified',
+    'params' => [
+      'version' => 4,
+      'values' => [
+        'name' => 'Mail_Accounts_Table_1',
+        'label' => E::ts('Mail Accounts Table 1'),
+        'saved_search_id.name' => 'Mail_Accounts',
+        'type' => 'table',
+        'settings' => [
+          'description' => NULL,
+          'sort' => [],
+          'limit' => 50,
+          'pager' => [],
+          'placeholder' => 5,
+          'columns' => [
+            [
+              'type' => 'field',
+              'key' => 'name',
+              'dataType' => 'String',
+              'label' => E::ts('Name'),
+              'sortable' => TRUE,
+            ],
+            [
+              'type' => 'field',
+              'key' => 'server',
+              'dataType' => 'String',
+              'label' => E::ts('Server'),
+              'sortable' => TRUE,
+            ],
+            [
+              'type' => 'field',
+              'key' => 'username',
+              'dataType' => 'String',
+              'label' => E::ts('Username'),
+              'sortable' => TRUE,
+            ],
+            [
+              'type' => 'field',
+              'key' => 'localpart',
+              'dataType' => 'String',
+              'label' => E::ts('Localpart'),
+              'sortable' => TRUE,
+            ],
+            [
+              'type' => 'field',
+              'key' => 'domain',
+              'dataType' => 'String',
+              'label' => E::ts('Domain'),
+              'sortable' => TRUE,
+            ],
+            [
+              'type' => 'field',
+              'key' => 'return_path',
+              'dataType' => 'String',
+              'label' => E::ts('Return-Path'),
+              'sortable' => TRUE,
+            ],
+            [
+              'type' => 'field',
+              'key' => 'protocol:label',
+              'dataType' => 'String',
+              'label' => E::ts('Protocol'),
+              'sortable' => TRUE,
+            ],
+            [
+              'type' => 'field',
+              'key' => 'source',
+              'dataType' => 'String',
+              'label' => E::ts('Mail Folder'),
+              'sortable' => TRUE,
+            ],
+            [
+              'type' => 'field',
+              'key' => 'is_ssl',
+              'dataType' => 'Boolean',
+              'label' => E::ts('Use SSL?'),
+              'sortable' => TRUE,
+            ],
+            [
+              'type' => 'html',
+              'key' => 'is_default',
+              'dataType' => 'Boolean',
+              'label' => E::ts('Used For'),
+              'sortable' => TRUE,
+              'rewrite' => "{if '[is_default]' == 'Yes'}" . E::ts("Bounce Processing <strong>(Default)</strong>") . "{else}" . E::ts("Email-to-Activity") . "{/if}",
```

Comment:
Oof, this will only work in English. This is another reason we need access to raw values in tokens.

Reply:
See #26742